src/System/GameDataFunction.h

```diff
@@ -0,0 +1,499 @@
+#pragma once
+
+#include <basis/seadTypes.h>
+#include <math/seadVectorFwd.h>
+#include <prim/seadSafeString.h>
+#include <nn/g3d/ResFile.h>
```

Comment:
forward-decl

Reply:
what exactly am i forward declaring?

Reply:
You should be forward-declaring `ResFile` instead of including the actual source.